charts/trento-server/values.yaml

```diff
@@ -37,6 +40,10 @@ postgresql:
   initdbScripts:
     init.sql: |
       CREATE DATABASE trento_event_store;
+      CREATE DATABASE wanda;
+
+wanda:
```

Comment:
I would put `wanda` just after `trento-runner`, to make it more prominent

Reply:
Agreed, I'd prefer to have our own components before the external dependencies